docs/about/privacy.md

```diff
@@ -0,0 +1,2 @@
+# Privacy Policy
+This site do not use any cookies or other tools to track user behavior. Server metrics such as number of HTTP requests will be logged, but no Personal Identifiable Information (PII) is collected.
```

Comment:
Simple and easy to understad. Except from a typo (do should be does) I think it looks good. Of course I am not a lawyer, but if you don't use cookies or track, you should be fine.

Reply:
great, thanks!